service/controller/resource/deployment/update_test.go

```diff
@@ -44,6 +45,7 @@ func Test_Resource_Deployment_newUpdateChange(t *testing.T) {
 		CurrentState                interface{}
 		DesiredState                interface{}
 		ExpectedDeploymentsToUpdate []*v1.Deployment
+		FakeTCObjects               []runtime.Object
```

Comment:
Need to also change the test name to mirror the change of which function is tested now.